glean_parser/templates/go_server.jinja2

```diff
@@ -182,28 +195,28 @@ func newGleanEvent(category, name string, extra map[string]string) gleanEvent {
 {% for event in events %}
 
 type {{ event|event_type_name }} struct {
-    {% for extra, metadata in event.extra_keys.items() %}
-    {{ extra|event_extra_name }} {{ metadata.type|go_metric_type }} // {{ metadata.description|clean_string }}
-    {% endfor %}
+	{% for extra, metadata in event.extra_keys.items() %}
```

Comment:
This template was mixed tabs & spaces, so this whitespace change is to get everything on tabs

Reply:
![](https://i.giphy.com/media/v1.Y2lkPTc5MGI3NjExOWhtcm44Z3RwbTNwOG9xMHJzcXc1YW1iYXJvaG9tczF0NGs1dHk0MyZlcD12MV9pbnRlcm5hbF9naWZfYnlfaWQmY3Q9Zw/LROKNqr26NG48/giphy.gif)